developer/src/kmcmplib/src/Compiler.cpp

```diff
@@ -3425,12 +3425,9 @@ KMX_DWORD ReadLine(FILE* fp_in , PKMX_WCHAR wstr, KMX_BOOL PreProcess)
 
   if (cur == fsize)
 
-    // Always a "\r\n" to the EOF, avoids funny bugs
-#if defined(_WIN32) || defined(_WIN64)
-  u16ncat(str, u"\r\n", _countof(str));  // I3481
-#else
+  // Always a "\r\n" to the EOF, avoids funny bugs
```

Comment:
Hmm, this comment now no longer matches what the next (uncommented) line does.

Reply:
This is to help to find the cause of "funny bugs" should they still occur after omitting "\r"